server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -840,16 +843,35 @@ private Optional<FormTag> addressCorrectionEnabledToggle(
                 + " feature can only be enabled once per screen."
             : " You can select one address question to correct per screen.";
 
-    if (!featureFlags.getFlagEnabled(request, ESRI_ADDRESS_CORRECTION_ENABLED)) {
-      toolTipText +=
-          " To use this feature, you will need to have your IT manager configure the GIS service.";
-    }
     if (questionIsUsedInPredicate) {
       toolTipText +=
           " Questions used in visibility or eligibility conditions must have address correction"
               + " enabled.";
     }
 
+    ATag learnMoreLink =
+        new LinkElement()
+            .setStyles("mb-2", "text-sm", "underline")
+            .setText("Learn more")
+            .setHref("https://docs.civiform.us/it-manual/sre-playbook/configure-gis-service")
+            .opensInNewTab()
+            .asAnchorText();
+    DivTag toolTip;
+    if (addressCorrectionDisabled) {
+      // Leave the space at the end, because we will add a "Learn more" link. This
+      // should always be the last string added to toolTipText for this reason.
+      toolTipText +=
+          " To use this feature, you will need to have your IT manager configure the GIS service. ";
+      // We must set mt-0, overriding mt-2.5 that comes before it in the classes string, or else
+      // a user will be unable to move their mouse inside the tooltip to be able to click on the
+      // link without the tooltip disappearing.
+      toolTip =
+          ViewUtils.makeSvgToolTip(
+              toolTipText, Icons.INFO, Optional.of("right-1/2 mt-0"), learnMoreLink);
```

Comment:
Not sure the `mt-0` is actually working reliably.  Hmm...

Reply:
Probably a bad idea to include both `mt-2.5` and `mt-0` anyway.  I think I'll just have to create a new function in ViewUtils instead.